source/manual/how-to-backup-and-restore-in-aws-rds.html.md

```diff
@@ -29,159 +24,102 @@ Before you get started you need to know:
 
 Using the stored variables from the previous steps:
 
 ```sh
-gds-cli aws govuk-${environment?}-admin \
-  aws rds restore-db-instance-from-db-snapshot \
-    --db-subnet-group-name ${db_subnet_group_name?} \
-    --db-instance-identifier restored-${db_instance_identifier?} \
-    --db-snapshot-identifier ${snapshot_arn?}
+aws rds restore-db-instance-from-db-snapshot \
+  --db-subnet-group-name $DB_SUBNET_GROUP_NAME \
+  --db-instance-identifier restored-$DATABASE_ID \
+  --db-snapshot-identifier $SNAPSHOT_ARN
 ```
 
 To see the newly created database instance, log into AWS Console > RDS > Databases > filter for your database name. You should see the original and newly created one.
 
-### 5. Test the database has been fully restored
+### 4. Test the database has been fully restored
 
 Before moving on to the next step we need to ensure that the database has been fully restored and is ready to be used:
 
 ```sh
-gds-cli aws govuk-${environment?}-admin \
-  aws rds wait db-instance-available \
-    --db-instance-identifier restored-${db_instance_identifier?}
+aws rds wait db-instance-available --db-instance-identifier restored-${DATABASE_ID}
 ```
 
 This command will wait until the database is ready, and then exit without any output.
 
 ### 6. Get the new database's hostname
 
-Once the database is ready, fetch its hostname:
+Once the database is ready, update the value of the SSM parameter
 
 ```sh
-gds-cli aws govuk-${environment?}-admin \
-  aws rds describe-db-instances \
-    --db-instance-identifier "restored-${db_instance_identifier?}" \
-    --query 'DBInstances[].Endpoint.Address'
+aws rds describe-db-instances \
+  --db-instance-identifier "restored-${DATABASE_ID}" \
+  --query 'DBInstances[].Endpoint.Address'
 ```
 
 Make a note of this.
 
-### 7. Connect to the restored backup database
+### 7. Update the existing secrets manager secret value
 
-This requires updating the `govuk/local-links-manager/postgres` secret in AWS Secrets Manager.
+This requires updating the existing secrets manager secret for the database you've just restored
 
-1. Log in to AWS in the correct environment: `gds aws govuk-${environment?}-admin -l`
-1. In AWS Secrets Manager, search for and click on `govuk/local-links-manager/postgres`.
+1. Log in to AWS in the correct environment: `development, staging or production`
+1. In AWS Secrets Manager, search for and click on the relevant secret
 1. Under the "Overview" tab, in the "Secret Value" section, select "Retrieve Secret Value".
 1. Make a note of the existing value, in case you need to revert the changes (for example, if performing a drill).
 1. Click "Edit", and replace the value of the "host" and "dbInstanceIdentifier" fields with the URL and identifier of the new database instance. Click "Save".
```

Comment:
```suggestion
1. Click "Edit", and replace the value of the `host` and `dbInstanceIdentifier` fields with the URL and identifier of the new database instance. Click "Save".
```

Unrelated to your PR at all but I'd love if these were inline `code` for readability. Thoughts?

Reply:
Totally fine to change that - I'll update

Reply:
Fixed